vpn/dns.go

```diff
@@ -0,0 +1,55 @@
+package vpn
+
+import "tailscale.com/net/dns"
+
+func NewVPNManager(t *Tunnel) dns.OSConfigurator {
```

Comment:
```suggestion
func NewDNSConfigurator(t *Tunnel) dns.OSConfigurator {
```

1. Go style typically eschews the package name from components. E.g. in `d := vpn.NewVPNConfigurator`, the second `VPN` doesn't give you much new information.
2. Let's stick with the Tailscale terminology and call this a "configurator" not a "manager".

Reply:
The `manager` usage came from Tailscale calling all their OSConfigurator implementation files `managers`, `manager_darwin`, etc. but sure enough they use `configurator` for all the public types.